docs/source.md

```diff
@@ -24,7 +24,7 @@
 3. Move the resulting `bin/gh` executable to somewhere in your PATH
 
    ```sh
-   $ sudo mv ./bin/gh /usr/local/bin/
+   $ sudo mv ./bin/gh /usr/bin/
```

Comment:
You probably don't want to put this into `/usr/bin`, instead you should add `/usr/local/bin` to your `$PATH` if it's missing.